### ISSUE TITLE
How to Access model history in tflearn to plot error?

### ISSUE THREAD

**Issue by warisha** [tflearn/tflearn#793]
i want to plot error using tflearn. I have generated same graph in keras with history but in tflearn i'm unable to do this. Kindly please help me!

**Comment by moumine9** [comment]
What do you mean by 'error', can you be more precise please ? I think you can use [Tensorboard](https://www.tensorflow.org/get_started/summaries_and_tensorboard) for different types of statistics. See whtat Tensorboard is, think about your errors and came back here if you need any help on how to use Tensorboard on TFLearn.

**Comment by warisha** [comment]
I want to plot training and validation loss like this. I want to plot such losses using tflearn

![image](https://user-images.githubusercontent.com/18666252/27088200-72df7818-5070-11e7-879a-7af2d7c5bb7b.png)

**Comment by moumine9** [comment]
Perfect. You can use [Tensorboard](http://tflearn.org/getting_started/#visualization) to that.
In order to use it, you have to manually setup in your code, which informations to be retrieved by adjusting the **tensorboard_verbose** variable. I recommend 3 (which makes your execution slower but gives you all the details regarding your network. For your case, `tensorboard_verbose = 0` is enough.

~~~ 
model = tflearn.DNN(network, checkpoint_path='/tmp/tflearn_logs/',max_checkpoints=1, tensorboard_verbose=0)
# By adjusting tensorboard_variable, we tell  Tflearn to save some datas that Tensorboard will master.

    model.fit(X,Y, validation_set=(Xtest,Ytest),n_epoch=10,show_metric=True, batch_size=64, snapshot_step=1, snapshot_epoch=True, run_id= 'name_modele')

# Remember, the run_id must be different for each execution so that you can retrieve a similar graph to what you have up'.
~~~

After the learning step has done, you can just open a terminal and enter the command below 
`$ tensorboard --logdir='/tmp/tflearn_logs'`

I recommend launching Tensorboard once the traning step is done, otherwise, it might make your computer a little bit slower. Also, the directory is cleaned each time the computer is rebooted. Make sure to save datas in the folder `/tmp/tflearn_logs` before turning it off.

**Comment by warisha** [comment]
Thanks, 
I want to ask one more thing  i am using TFLearn ResNet model for facial keypoints detection.

![2](https://user-images.githubusercontent.com/18666252/27231166-30fbd0d8-52cb-11e7-99f9-ae8f8860b2c3.png)

When i use this code to predict and plot 15 points on face, got this error. Can you help me to solve this problem?

![untitled](https://user-images.githubusercontent.com/18666252/27231205-4ad22f5c-52cb-11e7-867c-3996a0d00a1f.png)

**Comment by moumine9** [comment]
According to python, you have this error : ` TypeError : can only concatenante list (not int) to list `. As I cannot see you're code, I suggest you to randomly run some `print()` on each variable. I suspect you did not index properly your lists.
